examples/online_serving/openai_transcription_client.py

```diff
@@ -0,0 +1,23 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
nit: we should put the instructions to host the model with the server so this client code will work "out of the box"